README.md

```diff
@@ -0,0 +1,70 @@
+Rancher documentation is available at <http://docs.rancher.com/rancher/>
+Rancher OS documentation is available at <http://docs.rancher.com/os/>
+## Contributing to Rancher Labs Documentation Project
+### About Rancher Labs Documentation Site
+Rancher Lab documentation is hosted on Githubpages, and published it online by using Jekyll, an easy blog-aware static website generator. For more details on how to set up Jekyll, we recommend you to read <https://help.github.com/articles/using-jekyll-with-pages/>. If you are using Windows, we strongly advise you to follow the instruction given at <http://jekyllrb.com/docs/windows/>.
+For information on editing .md files (Markdown), refer to <https://daringfireball.net/projects/markdown/syntax>
+### Setting up the Git Environment
+In your browser, navigate to <https://github.com/rancher/rancher.github.io>
+Fork this repository by clicking the Fork button on the upper right-hand side. A fork of the repository is generated for you. On the right-hand side of the page of your fork, under 'HTTPS clone URL', copy the URL by clicking the Copy to clipboard icon.
+On your computer, follow these steps to setup a local repository to start working on the documentation:
+ 
+    $ git clone https://github.com/YOUR_ACCOUNT/rancher.github.io.git
+    $ cd rancher.github.io
+    $ git remote add upstream https://github.com/rancher/rancher.github.io
+    $ git checkout master
+    $ git fetch upstream
+    $ git merge upstream/master
+  
+### Updating the Files
+We recommend you to create a new branch to update the documentation files and that you do not disturb the master branch,  other than pulling in changes from upstream/master.
+For example, you create a branch, dev, on your computer to make changes locally to the documentation. This dev branch will be your local repository which then be pushed to your forked repository on GitHub where you will create a Pull Request for the changes to be committed into the official documentation.
+It is a healthy practice to create a new branch each time you want to contribute to the documentation and only track the changes for that pull request in this branch.
+
+    $ git checkout -b dev
+    (-b creates a new repository named dev. Now you can make necessary changes to the documentation)
+    $ git add .
+    $ git commit -a -m "commit message for your changes"
+    
+### Merging upstream/master into Your Local Branch (dev)
+Maintain an up-to-date master branch in your local repository. Merge the changes on a daily basis from the upstream/master (the official documentation repository) into your local repository. Ensure that you do complete this activity before you start working on a feature as well as right before you submit your changes as a pull request. You can also do this process periodically while you work on your changes to make sure that you are working off the most recent version of the documentation.
+
+    $ git checkout master
+    (Checkout your local master branch)
```

Comment:
Once it goes into the fenced code block, use a # sign in front of comments for code.